app/models/manageiq/providers/storage_manager/swift_manager/refresh_parser/cross_linkers.rb

```diff
@@ -5,6 +5,10 @@ def self.cross_link(ems, data)
       _log.warn "Manager does not have a parent."
       return
     end
+    unless data
+      _log.warn "Manager does not have any object storage."
```

Comment:
That's why I made it just a comment, not request changes.  We seem to be losing consistency on this one since the rubocop doesn't complain about it.